Userland/Libraries/LibMedia/CMakeLists.txt

```diff
@@ -4,15 +4,22 @@ set(SOURCES
     Color/TransferCharacteristics.cpp
     Containers/Matroska/MatroskaDemuxer.cpp
     Containers/Matroska/Reader.cpp
-    FFmpeg/FFmpegVideoDecoder.cpp
     PlaybackManager.cpp
     VideoFrame.cpp
 )
 
+if (NOT ANDROID)
+    list(APPEND SOURCES FFmpeg/FFmpegVideoDecoder.cpp)
+else()
+    list(APPEND SOURCES FFmpeg/FFmpegVideoDecoderStub.cpp)
```

Comment:
Any thoughts on what Video decoder we should use on android? Is this just a vcpkg thing for now, or does ffmpeg not support android at all?

Reply:
Can you make an issue to pick a suitable video decoder?

Reply:
FFmpeg built via vcpkg does support Android, so that is a possibility, but I'm not sure about the licensing issues mentioned here: https://github.com/LadybirdBrowser/ladybird/issues/254#issuecomment-2192903020

I made a new issue about it (#421) - I think best would be to use the Android NDK's Media API.